### DISCUSSION TITLE
Backup starting too soon after wakeup on laptop

### DISCUSSION THREAD

**Discussion by jeancf** [borgbase/vorta#2114] [FAQ] [answered]
I installed vorta + borg on my laptop and configured it to backup on a remote host. Manual backups work fine. As I don't use the laptop much, backups are mostly triggered as _missed backups_ when the laptop wakes up.

But most of the _missed backups_ fail with the following messages:
```
[Remote] Backup started.
[Remote] ERROR: Connection closed by remote host. Is borg working on the server?
```

This happens because vorta initiates the backup before the Wi-Fi connection has had the time to be reestablished. When I click on _Start Backup_ when I see that the connection is up, the backup executes without problem.

Is there a way to configure a delay before vorta attempts a backup after a wake-up?

**Comment by m3nu** [comment, marked as answer]
As a workaround, you could try a pre-backup command of `sleep 20`?

**Reply by jeancf** [reply]
It works, thanks. The only downside is that the UI is blocked while sleep is running but it is a minor annoyance.

**Comment by goebbe** [comment]
@jeancf What OS and Desktop environment are you using? 
There should be an easy way to add a sleep command in a non-UI-blocking way. 
E.g. on Linux / bash, you could try `sleep 20s && vorta &` to get the command de-syncronized.

**Comment by jeancf** [comment]
I am on Manjaro. 

Le 12 novembre 2024 09:12:40 GMT+01:00, goebbe ***@***.***> a écrit :

**Reply by goebbe** [reply]
> I am on Manjaro. Le 12 novembre 2024 09:12:40 GMT+01:00, goebbe ***@***.***> a écrit :

Which Desktop environment do you use?  (Xfce, Gnome, KDE?) or headless?

**Reply by goebbe** [reply]
EDIT: The discussion is about a running instance of Vorta (after sleep) - when the Wifi is not yet available. 
Since my answer was not helpful in this case - I decided to delete it - in order to avoid confusion.

**Comment by jeancf** [comment]
Sorry I was too quick to respond. I am on Manjaro KDE plasma 6.3. I have vorta started with autostart at the beginning of each session and staying alive in the system tray. I have added `sleep 20` in the Pre-backup field under Schedule > Shell Commands.

Just to be clear, it is on wake-up from sleep that the issue is present (i.e. when vorta is already running), not on login. I am not sure the solution you propose would help in the wake-up situation.

**Reply by goebbe** [reply]
Thanks for pointing that out - you are completely right. Sorry for the noise - I will edit my last comment, to reduce the confusion on this discussion.

**Reply by jeancf** [reply]
No worries at all. Thanks for taking the time to help.

**Comment by ctheune** [comment]
This happens to me, too and it seems to be the same as #1815
